suites/portal/registerUserTest.js

```diff
@@ -48,7 +48,7 @@ async function getAlltabs(I) {
     const disabledbt = '//*[@class="g3-dropdown-button__wrapper g3-dropdown-button__wrapper--disabled "]';
     const btPending = await tryTo(() => I.waitForElement(disabledbt, 2)); // eslint-disable-line
     if (!btPending) {
-      I.waitForClickable('//button[contains(text(),"Login to download")');
+      I.waitForClickable('//button[contains(text(),"Login to download")]');
```

Comment:
is it missing the opening `[`

Reply:
It was missing ], now it should be correct